extension-config/content-script-comms.js

```diff
@@ -0,0 +1,40 @@
+function toggleTota11y(payload, sendResponse) {
```

Comment:
Could you add a comment at the top of the file explaining what the file contents are for?